convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -318,54 +318,6 @@ def test_call_yum_cmd_with_repo_overrides(self):
             ],
         )
 
-    def test_get_pkgs_to_distro_sync(self):
-        problematic_pkgs = {
-            "protected": set(["a"]),
-            "errors": set(["b"]),
-            "multilib": set(["c", "a"]),
-            "required": set(["d", "a"]),
-            "mismatches": set(["e", "a"]),
-        }
-        all_pkgs = pkghandler.get_pkgs_to_distro_sync(problematic_pkgs)
-        self.assertEqual(
-            all_pkgs,
-            problematic_pkgs["errors"]
-            | problematic_pkgs["protected"]
-            | problematic_pkgs["multilib"]
-            | problematic_pkgs["required"],
-        )
-
-    @unit_tests.mock(pkghandler, "call_yum_cmd", CallYumCmdMocked())
-    def test_resolve_dep_errors_one_downgrade_fixes_the_error(self):
-        pkghandler.call_yum_cmd.fail_once = True
-
-        pkghandler.resolve_dep_errors(YUM_PROTECTED_ERROR, set())
```

Comment:
You can remove all the constants used by the removed tests on lines [1741-1843](https://github.com/oamg/convert2rhel/pull/822/files#diff-c3ce6b0e1e7ddae0678f9eb7c71f9c5f8aecbfd5b7db1deb7d8947ccf1492a08L1741-L1843).

Reply:
good catch! will remove